### ISSUE TITLE
[vim-mode]: Allow :W to write file, and :Wa to write all files

### ISSUE THREAD

**Issue by Tobbe** [zed-industries/zed#13826]
### Check for existing issues

- [X] Completed

### Describe the feature

Whenever I want to save the current file I use vim's `:w` command. The problem is I often make a type and type `:W` instead (not releasing the shift key fast enough). Same for `:wa` which becomes `:Wa`.

I'd like a way to remap `:W` to `:w`, `:Wa` to `:wa`, `:Wq` to `:wq` etc

A feature that would just let me hard-code the variants I need are good enough for me, like 
```
command! -bang -range=% -complete=file -nargs=* W <line1>,<line2>write<bang> <args>
```
(from https://stackoverflow.com/questions/10590165/is-there-a-way-in-vim-to-make-w-to-do-the-same-thing-as-w)

There was a brief discussion of this feature request here: https://discord.com/channels/869392257814519848/1212320652837789707/1258393771033493536

**Comment by ConradIrwin** [comment]
TBH, I think we should build these ones in; but at some point command palette aliases would be nice.

If you want to pair with me on this: https://calendly.com/conradirwin/pairing.

The current match statement isn't particularly elegant (https://github.com/zed-industries/zed/blob/5256620aeefb36db5aa168fb865cd5a1034b3d5b/crates/vim/src/command.rs#L155) so it seems fine to just add more things to it (and at some point we should refactor)

**Comment by Tobbe** [comment]
@ConradIrwin Thanks for the invitation! I'd love to pair with you on that 😀 
I booked a time on your calendar for next week

**Comment by JonGretar** [comment]
This should probably be marked closed as #14826 implemented this back in july. :)
